src/Compilers/Core/Portable/SourceGeneration/GeneratorTimerExtensions.cs

```diff
@@ -42,6 +42,22 @@ public static RunTimer CreateSingleGeneratorRunTimer(this CodeAnalysisEventSourc
             }
         }
 
+        public static void PooledWhenSparse(this CodeAnalysisEventSource eventSource, Type type, int count, int capacity)
```

Comment:
>PooledWhenSparse [](http://example.com/codeflow?start=27&length=16)

Are there any callers of these two methods?

Reply:
these methods are gone now :)